client/tests/util/getPage.js

```diff
@@ -113,9 +125,17 @@ const getPage = async (options, callback) => {
         throw new Error('Please provide a URL, even if it it is simply "/"');
     }
 
-    await page.goto(`http://localhost:${CLIENT_PORT}${url}`);
+    if (isDebugMode) {
+        page.setDefaultTimeout(604800000 /* a week should be enough */);
```

Comment:
> a week should be enough 

I would hope so! :)

Reply:
Me too haha 😂 some bugs seem like a week would be necessary.